stdlib/string.gr

```diff
@@ -1769,27 +1913,25 @@ let trimString = (str: String, end: Bool) => {
   let chars = explode(str), charsLength = length(str)
   let mut i = 0, offset = 1
   if (end) {
-    i = charsLength-1
+    i = charsLength - 1
     offset = -1
   }
   for (; i < charsLength && i > -1; i += offset) {
-    let currentChar = chars[i];
+    let currentChar = chars[i]
     // TODO: Use unicode whitespace property and unicode line terminator once github issue #661 is completed
     if (
       // Spacing
-      currentChar != '\u{0009}'  && // Tab
-      currentChar != '\u{000B}'  && // LINE TABULATION
-      currentChar != '\u{000C}'  && // FORM FEED (FF)
-      currentChar != '\u{0020}'  && // Space
-      currentChar != '\u{00A0}'  && // No Break Space
-      currentChar != '\u{FEFF}'  && // ZERO WIDTH NO-BREAK SPACE
-      // Line Terminators
-      currentChar != '\n' && // LF
+       currentChar != '\u{0009}' && currentChar != '\u{000B}' &&
+      currentChar != '\u{000C}' &&
+      currentChar != '\u{0020}' &&
+      currentChar != '\u{00A0}' &&
+      currentChar != '\u{FEFF}' &&
+      currentChar != '\n' &&
```

Comment:
This seems wrong.

Reply:
I need to ask a question about those comparison operators...

Reply:
Fixed.